src/features/movies/MovieList/MovieTile/styled.js

```diff
@@ -1,14 +1,38 @@
 import styled from "styled-components";
-import { ReactComponent as Star } from "../../../../images/star.svg";
+import {ReactComponent as Star} from "../../../../images/star.svg";
+import camera from "../../../../images/camera.svg";
 
 export const Content = styled.li`
+    cursor: pointer;
     padding: 16px;
     background-color: ${({ theme }) => theme.color.white};
     box-shadow: 0 4px 12px rgba(186, 199, 213, 0.5);
     border-radius: 5px;
-    display: flex;
-    flex-direction: column;
     transition: 0.2s;
+    display: grid;
+    justify-content: center;
+    grid-template-columns: 1fr;
```

Comment:
Oo, i missed out that 
i added 
`grid-template-rows: max-content repeat(3, min-content);`
max contetn for poster, 3 rows for title year and tags, and empty and rating is deafult 1fr and its work perfect

Reply:
Notice that rating container is not aligned through all the tiles - in some it is higher and in some lower. It is because this row is not always the same height - it should not be 1fr.

Reply:
`grid-template-rows: max-content repeat(3, min-content) 1fr;`
now problem resolved